src/qz/installer/MacInstaller.java

```diff
@@ -12,6 +12,7 @@
 import org.apache.commons.lang3.StringUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import qz.common.Constants;
```

Comment:
Aren't we already static importing this?

Reply:
Yes, thanks.